shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -199,7 +199,22 @@ async fn cluster_single_rack_v4(#[case] driver: CassandraDriver) {
 
     let shotover_manager =
         ShotoverManager::from_topology_file("example-configs/cassandra-cluster/topology-v4.yaml");
-    cluster::single_rack_v4::test_node_going_down(compose, shotover_manager, driver).await;
+    cluster::single_rack_v4::test_node_going_down(compose, shotover_manager, driver, false).await;
+}
+
+#[cfg(feature = "cassandra-cpp-driver-tests")]
+#[rstest]
+//#[case::cdrs(CdrsTokio)]
+#[cfg_attr(feature = "cassandra-cpp-driver-tests", case(Datastax))]
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn cluster_single_rack_node_lost(#[case] driver: CassandraDriver) {
+    let compose =
+        DockerCompose::new("example-configs/cassandra-cluster/docker-compose-cassandra-v4.yaml");
+
+    let shotover_manager =
+        ShotoverManager::from_topology_file("example-configs/cassandra-cluster/topology-v4.yaml");
+    cluster::single_rack_v4::test_node_going_down(compose, shotover_manager, driver, true).await;
```

Comment:
Couldn't this test just go at the end of the other tests?

Reply:
which test are you suggesting? I dont think any of the other existing tests are compatible with it?

Reply:
I was suggesting it could go the end of the `single_rack_v4` test.

Reply:
![image](https://user-images.githubusercontent.com/5120858/199652459-52ecf7c5-0c11-4b54-bb5e-2716e08882b9.png)
From the PR description

Reply:
Yes but if it goes as the last test it won't matter?

Reply:
There is already a test_node_going_down call at the end of single_rack_v4.
We cant have 2 test_node_going_down calls at the end of single_rack_v4 because the first call would leave the cluster in a state where the 2nd call would fail

Reply:
Ah my mistake